File: ecmascript/src/fizz-buzz.js
'use strict'

const _ = require('lodash');

const fizzBuzz = function() {
    console.log('Starting Fizz Buzz');
    for (const idx of _.range(1, 100)) {
        if (idx % 15 === 0) {
            console.log('FizzBuzz');
        } else if (idx % 3 === 0) {
            console.log('Fizz');
        } else if (idx % 5 === 0) {
            console.log('Buzz');
        } else {
            console.log(idx);
        }
    }
}

module.exports = fizzBuzz;

File: c-sharp/FizzBuzz.cs
using System;
using System.Linq;

public partial class Koan
{
    public void FizzBuzz()
    {
        Console.WriteLine("Staring FizzBuzz!");
        foreach(var idx in Enumerable.Range(1, 100))
        {
            switch (idx)
            {
                case int x when x % 3 == 0:
                    Console.Write("Fizz");
                    break;
                case int x when x % 5 == 0:
                    Console.Write("Buzz");
                    break;
                default:
                    Console.Write(idx);
                    break;
            }
            Console.Write(Environment.NewLine);
        }
    }
}